aioamqp/protocol.py

```diff
@@ -139,10 +139,17 @@ def start_connection(self, host, port, login, password, virtualhost, ssl=False,
         """Initiate a connection at the protocol level
             We send `PROTOCOL_HEADER'
         """
-
-        if login_method != 'AMQPLAIN':
-            # TODO
-            logger.warning('only AMQPLAIN login_method is supported, falling back to AMQPLAIN')
+        if login_method == 'EXTERNAL':
```

Comment:
Just a side note here, probably not worth changing current code right now, an `Enum` would probably be nice to have here at some point.